code/persian/index.ts

```diff
@@ -0,0 +1,58 @@
+import { build, transform } from '~/base'
+
+const consonants = {
+'\u0621': 'ʔ ',
```

Comment:
There are spaces in here, there must be no spaces.

Reply:
Also, @DhruboSaeed did you run `node run move:talk` or `pnpm move:talk`?

Reply:
Also, please run `pnpm install` on the repo, so you can get the clean on save functionality.

Reply:
@DhruboSaeed did you do this step yet too?

Reply:
If you don't have `pnpm` installed, then use:

```
npm install -g pnpm
```

And you will get it!

Reply:
I have remove the whitespace in the value string and install the pnpm at repo. 
didnot know why it is showing all red. 
![Image1](https://github.com/user-attachments/assets/2aaa3ddd-e9e0-4578-b013-57876732593b)